source/server/converters.h

```diff
@@ -335,6 +335,12 @@ inline ::grpc::Status ApiErrorAndDescriptionToStatus(int32_t status, std::string
   nlohmann::json jsonError;
   return ApiErrorAndDescriptionToStatus(status, description, jsonError);
 }
+
+inline ::grpc::Status ApiErrorToStatus(int32_t status)
```

Comment:
I added this to handle the cases where the function is marked with 'exclude_from_get_last_error'. I think populating the error status with a description of "Unknown" would be alright here and at least consistent with the other APIs that return an UNKNOWN gRPC status.